walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,154 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optionalAttendeeShouldNotImpactSchedule() {
+    // Have each person have different events. We should see two options because each person has
```

Comment:
These comments aren't terribly helpful since they are often the same on different tests. Either make them more relevant to the individual tests or remove them. I think removing is fine in many cases since the diagrams can be pretty self-explanatory.